README.md

```diff
@@ -16,17 +17,17 @@ A new Flutter project
 
 - Simple UI for food delivery app's
 
-- use SQLite for store order data in local database 
+- Use SQLite to store order data in a local database 
```

Comment:
the previous sentence was also right, 

but anyway Thanks for the improvement!